modules/local/sentieon/dnascope.nf

```diff
@@ -0,0 +1,60 @@
+process SENTIEON_DNASCOPE {
+    tag	"$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path fasta
+    path fai
+    path known_dbsnp
+    path known_dbsnp_tbi
+    path ml_model
+    val pcrfree
```

Comment:
to be removed

Reply:
First I wanted to pass it as an input variable then thought maybe it would be better to have  it as a parameter and added the variable in params section where we can add a bit of description in  module.config but forgot to remove it here. 
So args3 actually returns this.

Reply:
Haven't pushed it yet though

Reply:
Oh wait, you only wanted to remove line 13?

Reply:
yes, added  it in nextflow_schema.json and module.config